src/sql/connection.py

```diff
@@ -41,7 +41,7 @@ def set(cls, descriptor):
                            cls.connections.get(descriptor.lower())
             cls.current = existing or Connection(descriptor)
         else:
-            if cls.connections:
+            if cls.connections and verbose:
```

Comment:
This causes the default connection to not be used if you have disabled feedback. You need another check to see if `cls.current` is defined and only go to the environment if it is not.